RNA-seq/setup/magic-files.md

```diff
@@ -8,8 +8,20 @@ The sub-path for each file should be the same as that for `training-modules/RNA-
 
 Files are listed below by the notebook that produces them:
 
+- `02-gastric_cancer_tximeta.Rmd`
+  - `data/gastric-cancer/txi/gastric-cancer_tximeta.rds`
 - `setup/ref_notebooks/04-nb_cell_line_tximeta.Rmd` (participants create their own notebook for this step)
   - `data/NB-cell/txi/NB-cell_tximeta.rds`
 - `05-nb_cell_line_DESeq2.Rmd`
   - `results/NB-cell/NB-cell_DESeq_amplified_v_nonamplified.rds`
   - `results/NB-cell/NB-cell_DESeq_amplified_v_nonamplified_results.tsv`
+
+
+
+Note that the output files from `scripts/run_SRR585570.sh` script are already present at:
+
+- `/shared/data/training-modules/RNA-seq/data/salmon-quant/SRR585570/`
```

Comment:
This is not right and also I don't think should be here since it isn't output anyways (but in theory could be the gastric cancer input fastq?)
```suggestion
```

Reply:
or if it IS right, the folder isn't there.

Reply:
It's missing `gastric-cancer`: `/shared/data/training-modules/RNA-seq/data/gastric-cancer/salmon-quant/SRR585570/`

Reply:
Thanks for catching the path error. It should be 
```suggestion
- `/shared/data/training-modules/RNA-seq/data/gastric-cancer/salmon_quant/SRR585570/`
```